src/shared/check_outdated_data.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+int Run(int maxTimeDiff, char *aavild_data) {
+
+    time_t cur_time; 
+    //char * aavild_data = "20211122080000";
+    struct tm timeptr;
+
+    cur_time = time(NULL);
+    printf("Current timestamp = %ld\n", cur_time);
+
+    strptime(aavild_data, "%Y%m%d%H%M%S", &timeptr);
+    time_t aav_time = mktime(&timeptr);
+    printf("Aavild timestamp = %ld\n", aav_time);
+
+    double timediff = difftime(cur_time, aav_time);
+    printf("Difference between timestamps = %.0lf\n", timediff);
```

Comment:
Why are we printing here?

Reply:
We are printing the difference between the receiving data and the current data. There are no need to print, but it was just for me. I will make the printing part a comment.